PROJECT_GUIDELINES.md

```diff
@@ -0,0 +1,173 @@
+# Project Guidelines
+
+Hey there and thanks for stopping by! We welcome all feedback and contributions to Calcite App Components. If you are interested in getting involved, we ask that you give our guidelines a quick read through. Below is a list of things we've found work well for us as we continue developing our components.
+
+## TOC
+
+- [Code of Conduct](https://github.com/Esri/contributing/blob/master/CODE_OF_CONDUCT.md)
+- [Contibution Guidelines](https://github.com/esri/contributing)
+
+- [Conventions](#conventions)
+  - [Formatting](#formatting)
+  - [Github](#github)
+- [Dist/Package](https://github.com/Esri/calcite-app-components/blob/master/GETTING_STARTED.md_)
+- [Code Base](#code-base)
+  - [A11y](#a11y)
+  - [Components](#components)
+
+## Conventions
+
+### Formatting
+
+There is a script that automatically formats our code for us to keep everything looking clean, but we ask that you keep in mind things like formatting and other best practices when contributing to our repo.
+
+### Github
+
+#### Issues
+
+We have created [templates](https://github.com/Esri/calcite-app-components/issues/new/choose) for new issues for everyone to follow for consistency.
+
+#### Commits
+
+We ask that commits have informative headers and if necessary, a detailed body- squash-merge commit messages go straight into the changelog. Please follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)! If the contribution will cause a breaking change, be sure to add sufficient information about it.
+
+#### Pull Requests
+
+This project has an GitHub App installed to automatically perform semantic status checks on open pull requests.
+
+The [Semantic Pull Request](https://github.com/probot/semantic-pull-requests) status check will ensure your pull requests are semantic before you merge them.
+
+#### Be Semantic!
+
+- Add a semantic [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/).
+- Edit the PR title by adding a semantic prefix like `fix:` or `feat:` or any other [conventional commit type](https://github.com/commitizen/conventional-commit-types/blob/master/index.json).
+
+#### Contributing
+
+Please see our [contributing guidelines](https://github.com/Esri/calcite-app-components/blob/master/CONTRIBUTING.md).
+
+## Code Base
+
+Our code base is written in typescript.
+
+### A11y
+
+We generally follow the guidelines and standards in these articles:
+
+- [Google accessibility overview](https://developers.google.com/web/fundamentals/accessibility/)
+- [WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)
+
+### Components
+
+Have an idea for a new component? Awesome! Check out our [new component checklist](https://github.com/Esri/calcite-app-components/wiki/New-Component-Checklist) and submit an [issue](https://github.com/Esri/calcite-app-components/issues/new?assignees=&labels=new+component%2C+0+-+new%2C+architecture&template=new-component.md&title=New+Component%3A+).
```

Comment:
Before the checklist, new components should have an issue so we can determine whether they belong here or `calcite-components`. If a component does belong here, that's where the checklist can be used to guide development.